server/controllers/entities/lib/build_popularity_by_uri.js

```diff
@@ -77,12 +77,23 @@ const getEntitiesPopularityTotal = uris => {
   .then(results => _.sum(results) + results.length)
 }
 
+const getTypeAgnosticEntityScore = async uri => {
```

Comment:
That can be useful indeed (and fast). 
A naming proposition since those entities have a P31, and the client does categorize those with a type derived from this wikidata's P31. While what differs from other popularity builders is the fact that it doesnt query any entity graph, isn't it ? What about `getSimpleEntityScore` or `getChildAgnosticEntityScore`

[non-blocking]

Reply:
renamed to `getSimpleEntityScore` in 10a527607